app/models/concerns/feed_image_file.rb

```diff
@@ -4,7 +4,7 @@ module FeedImageFile
   extend ActiveSupport::Concern
 
   included do
-    belongs_to :feed, touch: true
+    belongs_to :feed, touch: true, optional: true
```

Comment:
this `optional: true` is another Rails default change, otherwise it validates the parent is not nil